### PR TITLE
Create bsendas.md

### PR DESCRIPTION
**Pull request by bsendas**

### CODE REVIEW

**Comment by bsendas** [comment]
@wulfland I know what I did wrong, I think now it is all 💯

File: docs/_slides/bsendas.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Greetings from @bsendas!
```
Comment (wulfland) [inline docs/_slides/bsendas.md:6]:
```suggestion
Greetings from @bsendas and @wulfland!
```

File: docs/_slides/bsendas.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+![dinotocat](https://octodex.github.com/images/dinotocat.png)
+{: .center}
```
Comment (wulfland) [inline docs/_slides/bsendas.md:9]:
```suggestion
![filmtocat](https://octodex.github.com/images/filmtocats.png)
{: .center}
```

File: docs/_slides/bsendas.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Greetings from @bsendas and @wulfland!
+
```
Comment (wulfland) [inline docs/_slides/bsendas.md:7]:
```suggestion

Greetings from @bsendas and @wulfland! Hello World

```

File: docs/_slides/bsendas.md [hunk 4]
```diff
@@ -3,9 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Add a text here to caption the image. Pick an image from [Octodex](https://octodex.github.com) to individualize your slide.
+Greetings from @bsendas and @wulfland! Hello World
```
Comment (wulfland) [inline docs/_slides/bsendas.md:6]: 👍

**Review by wulfland — APPROVED** [review]

**Pull request merged**